tensorflow-stubs/__init__.pyi

```diff
@@ -35,97 +81,159 @@ class dtypes:
 class Tensor:
     pass
 
+
 class gpu_options:
     def __init__(self):
         self.allow_growth: bool
 
+
 class ConfigProto:
     def __init__(self, log_device_placement: bool) -> None:
         self.gpu_options: gpu_options
-        #self.gpu_options.allow_growth: bool
+        # self.gpu_options.allow_growth: bool
+
 
 class Graph:
     pass
 
+
 class BaseSession:
-    #TODO: options is of type RunOption, run_metadata is of type RunMetadata
+    # TODO: options is of type RunOption, run_metadata is of type RunMetadata
     # Return type is option of:
     # single graph element if fetches is a single graph element  OR
     # list of graph elements if fetches is a list of single graph elements OR
     # a dictionary
     # Leaving it as Any for now
-    def run(self, fetches: Any, feed_dict: Optional[Dict[Any, Any]] = None, run_options: Any = None, run_metadata: Any = None) -> Any: ...
+    def run(self, fetches: Any,
+            feed_dict: Optional[Dict[Any, Any]]=...,
+            run_options: Any=...,
+            run_metadata: Any=...
+            ) -> Any: ...
```

Comment:
I don't think I like throwing away the information about what the default values are. I see that the defaults were replaced with ellipsis (`...`) in these